### DISCUSSION TITLE
zh/games/sokoban

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#18] [Q&A]
# zh/games/sokoban

高难度推箱子游戏，自带各种难度的关卡，也支持自定义游戏关卡并分享好友一起玩。动动脑子，把箱子绕过障碍物，推到指定的位置。益智游戏，老少皆宜，锻炼思维能力。

https://gallery.selfboot.cn/zh/games/sokoban?level=1

**Comment by weiyinfu** [comment]
giscus是啥

**Reply by selfboot** [reply]
就你用的这个评论组件
